src/gpu/intel/ocl/micro_sdpa.cl

```diff
@@ -33,6 +33,8 @@
 #define sg_per_wg (ugemm_kq_sg_per_wg_m * ugemm_kq_sg_per_wg_n)
 #define q_tile_sg_n DIV_UP(ugemm_kq_wg_tile_n, sg_per_wg)
 
+typedef enum { undefined, bottom_right, top_left } dnnl_causal_mask_type;
```

Comment:
How is it synchronized with external definition?

Reply:
Its not. Should I put the enum in its own file and reuse the header in the host and OpenCL code?